Sources/Widgets/Widgets3D/ResliceCursorWidget/behavior.js

```diff
@@ -318,31 +316,27 @@ export default function widgetBehavior(publicAPI, model) {
     vtkMath.normalize(cross);
 
     const sign = vtkMath.dot(cross, planeNormal) > 0 ? -1 : 1;
-
-    const matrix = mat4.create();
-    mat4.translate(matrix, matrix, center);
-    mat4.rotate(matrix, matrix, rotationAngle * sign, planeNormal);
-    mat4.translate(matrix, matrix, [-center[0], -center[1], -center[2]]);
+    const radianAngle = rotationAngle * sign;
 
     // Rotate associated line's plane normal
     const planeName = activeLine.getPlaneName();
     const normal = model.widgetState[`get${planeName}PlaneNormal`]();
-    const newNormal = vtkMath.rotateVector(
-      normal,
-      planeNormal,
-      rotationAngle * sign
-    );
+    const newNormal = vtkMath.rotateVector(normal, planeNormal, radianAngle);
     model.widgetState[`set${planeName}PlaneNormal`](newNormal);
 
     if (model.widgetState.getKeepOrthogonality()) {
       const associatedLineName = getAssociatedLinesName(activeLine.getName());
       const associatedLine = model.widgetState[`get${associatedLineName}`]();
       const planeName2 = associatedLine.getPlaneName();
       const normal2 = model.widgetState[`get${planeName2}PlaneNormal`]();
+
+      // Compute previous angle between lines
+      const previousAngle = vtkMath.angleBetweenVectors(normal, normal2);
+
       const newNormal2 = vtkMath.rotateVector(
-        normal2,
+        newNormal,
         planeNormal,
-        rotationAngle * sign
+        previousAngle
       );
       model.widgetState[`set${planeName2}PlaneNormal`](newNormal2);
     }
```

Comment:
Please fix also Widget/ResliceCursor (in addition to your Widget3D/ResliceCursor). Search in the code: `rotateVectorAboutVector`